### ISSUE TITLE
Review some of the documents 

### ISSUE THREAD

**Issue by yowill** [18F/dos-ops-center#40]
As an 18F Researcher, I want to get copies of the various documents that have been referenced recently so that I can better understand them and ask follow up questions.  (Not exhaustive but if people have other tangible lists or documents, add them to the issue.)

Background:
https://github.com/18F/dos-ops-center/blob/master/Editor%20Guidance.onepkg?raw=true

AC:
Get copies of and review:
- [ ] NIST Validated Product List
- [x] List of approved hardware and software - Information Technology Configuration control board
- [ ] Pick list of approved scripts
- [x] The Guidance (on github https://github.com/18F/dos-ops-center/blob/master/Editor%20Guidance.onepkg?raw=true)
- [ ] Potentially a template of a MemCon or redacted version (Tyler looking into)
- [x] Foreign Affairs Manual

**Comment by beebouttw** [comment]
[https://app.zenhub.com/files/154224183/75540e5e-8977-4c61-9bda-53194d937b66/download](https://app.zenhub.com/files/154224183/75540e5e-8977-4c61-9bda-53194d937b66/download)

ITCCB Approved Product List

**Comment by beebouttw** [comment]
Foreign Affairs Manual:  https://fam.state.gov/

**Comment by karareinsel** [comment]
Still waiting on NIST Validated Product List, Pick list of approved scripts, and  Potentially a template of a MemCon or redacted version. Recommend closing and creating a new issue for the 3 outstanding documents.
